vita3k/gui/src/vita3k_update.cpp

```diff
@@ -43,6 +42,8 @@ enum Vita3kUpdate {
 static Vita3kUpdate state;
 static VitaAreaState vita_area_state;
 
+static const auto latest_link = "https://api.github.com/repos/Vita3K/Vita3K/releases/latest";
+
 static int git_version;
 static std::vector<std::pair<std::string, std::string>> git_commit_desc_list;
 bool init_vita3k_update(GuiState &gui) {
```

Comment:
Since you are downloading updated Vita3K in `emuenv.base_path`, need to change path where checking existing file for macOS. Also need change in update script for macOS.

Reply:
script just run root, so no important